src/main/java/com/palantir/docker/compose/DockerComposition.java

```diff
@@ -110,9 +110,9 @@ public static DockerCompositionBuilder of(DockerCompose executable) {
         return new DockerCompositionBuilder(executable);
     }
 
-    public void exec(DockerComposeExecOption option, String containerName, DockerComposeExecArgument argument)
+    public void exec(DockerComposeExecOption dockerComposeExecOption, String containerName, DockerComposeExecArgument dockerComposeExecArgument)
             throws IOException, InterruptedException {
-        dockerCompose.exec(option, containerName, argument);
```

Comment:
I think the old version was probably better, but they do all need plurals, we're passing multiple options an arguments bundled into a single argument.